pkg/kubelet/kubelet_pods.go

```diff
@@ -161,6 +162,7 @@ func (kl *Kubelet) getKubeletMappings() (uint32, uint32, error) {
 	if string(outUids) != string(outGids) {
 		return 0, 0, fmt.Errorf("mismatched subuids and subgids for user %q", kubeletUser)
```

Comment:
since we are touching it, do we need to add the `user namespaces:` prefix?

Reply:
Idem!